src/Talis/Persona/Client/Base.php

```diff
@@ -55,8 +57,19 @@
     private $defaultTtl;
 
     /**
+     * @var string
+     */
+    private $appUserAgent;
+
+    /**
+     * @var string
+     */
+    private $phpVersion;
+
+    /**
      * Constructor
      *
+     * @param string $appUserAgent Consuming application user agent string @since 2.0.0
```

Comment:
I think it's worth putting some examples of valid user agent strings here (maybe what's not valid, too) to help encourage best practices.